build.gradle

```diff
@@ -175,3 +175,7 @@ subprojects {
 apply from: rootProject.file('gradle/compiler-version.gradle')
 apply from: rootProject.file("gradle/dokka.gradle")
 apply from: rootProject.file("gradle/benchmark-parsing.gradle")
+
+subprojects {
```

Comment:
Such clause will also affect utility projects like `benchmarks` and `guide`. Although it's not a big deal (they probably don't even have jar task), logically module configuration can belong here, where publishing itself is configured: https://github.com/Kotlin/kotlinx.serialization/blob/f7a7141fd8c6080484a1b37b5ada2525cf28369e/build.gradle#L154

Reply:
Yeah, those 2 projects don't have a `Jar` task so nothing happened there.
I moved the clause anyway, so it is called at a more logical place.